uitools/widgets/Esri/ArcGISRuntime/Toolkit/BasemapGallery.h

```diff
@@ -0,0 +1,40 @@
+#ifndef ESRI_ARCGISRUNTIME_BASEMAPGALLERY_H
+#define ESRI_ARCGISRUNTIME_BASEMAPGALLERY_H
+
+#include <QFrame>
+#include "Basemap.h"
+#include "BasemapGalleryController.h"
+namespace Ui {
+class BasemapGallery;
+}
+
+namespace Esri {
+namespace ArcGISRuntime {
+namespace Toolkit {
+class BasemapGalleryController;
+
+class BasemapGallery : public QFrame
+{
+  Q_OBJECT
+
+public:
+  explicit BasemapGallery(GeoModel* basemap, QWidget *parent = nullptr);
```

Comment:
🔧  Add a `Q_INVOKABLE` here. It allows the Qt metatype system to create a `BasemapGallery`.